public/style.css

```diff
@@ -13,16 +13,7 @@ li {
 }
 
 #user {
-  font-family: 'Barrio', cursive;
-  color: white;
   display: flex;
-  width: 100%;
-  height: 100%;
-  justify-content: center;
-}
-
-#user>div {
-  margin: auto;
 }
 
 #user .home-label {
```

Comment:
Can you remove this `#user` from here, I think it is useless.

Reply:
`39-42` done. About `15-17` already told you [here](https://github.com/KamandPrompt/baat-cheet/pull/95#issuecomment-704129610). Also removed `#userN` style and added bootstrap classes.